axum/src/routing/method_routing.rs

```diff
@@ -835,12 +835,11 @@ where
             S: Clone,
         {
             if endpoint_filter.contains(filter) {
-                if out.is_some() {
-                    panic!(
-                        "Overlapping method route. Cannot add two method routes that both handle \
+                assert!(
+                    !out.is_some(),
+                    "Overlapping method route. Cannot add two method routes that both handle \
                          `{method_name}`",
```

Comment:
Does this fit the line as a single line string literal now? If no, please reduce indentation on the second line by 4 spaces.